packages/@aws-cdk/aws-lambda-python/lib/types.ts

```diff
@@ -15,6 +15,13 @@ export interface BundlingOptions extends DockerRunOptions {
    */
   readonly poetryIncludeHashes?: boolean;
 
+  /**
+ * When using Poetry bundler, list of file patterns to exclude when copying assets for bundling.
+ *
+ * @default - Empty list
+ */
+  readonly poetryAssetExcludes?: string[];
```

Comment:
Is there a reason this should be scoped to `poetry` only and not just be called `assetExcludes`?

Reply:
Honestly, I was just trying to minimize the blast radius of this PR. But there is no real reason why this should be limited to `poetry` only. I can make the change to support all the packagers.

Seems like there might be existing issues that this PR could help out with too https://github.com/aws/aws-cdk/issues/23313

Reply:
You could definitely add support for that if you wanted, but this PR is also fine in its current scope 🙂

Reply:
I just pushed some new changes moving it to just `assetExcludes`. I will definitely need help verifying and cleaning up these integration tests, that seemed to have had a bit more of an impact adding this to the other tests 👀